test/test_meta.py

```diff
@@ -891,7 +922,25 @@ def test_meta(self, device, dtype, op):
     @skipIfCrossRef
     @suppress_warnings
     @ops(op_db)
-    def test_dispatch_meta(self, device, dtype, op):
+    def test_meta_inplace(self, device, dtype, op):
+        func = op.get_inplace()
+        if not func:
+            self.skipTest("No inplace variable for this op")
+        func = self._get_safe_inplace(func)
+        samples = op.sample_inputs(device, dtype, requires_grad=False)
+        for sample_input in samples:
+            if sample_input.broadcasts_input:
+                continue
+            args = [sample_input.input] + list(sample_input.args)
+            kwargs = sample_input.kwargs
+            with MetaCrossRefFunctionMode(self, dtype=dtype, device=device, inplace=True):
+                expected = func(*args, **kwargs)
```

Comment:
do we need this many variants of the test method?

Reply:
at least dedupe the logic?

Reply:
I do think we want different inplace variants so that inplace testing will not interact with the out of place one. For example if the automatic disable/flaky test skips this, it will be based on the test name. And so the inplace test can be disabled independently of the out of place one.

I will refactor the logic to reduce duplication now that CI is good yes.